crates/bevy_utils/src/synccell.rs

```diff
@@ -7,6 +7,7 @@
 ///
 /// Provides a wrapper that allows making any type unconditionally [`Sync`] by only providing mutable access.
 #[repr(transparent)]
+#[derive(Clone, Copy)]
 pub struct SyncCell<T: ?Sized> {
```

Comment:
I'd like extra attention paid here to make sure that it's sound to clone a `SyncCell`. I don't see why this would be unsound, but I also can't figure out how to prove that it *is* sound.